extensions/wikia/CommunityPage/tests/WikiTopicTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+class WikiTopicTest extends WikiaBaseTest {
+
+	public function setUp() {
+		$this->setupFile = __DIR__ . '/../CommunityPage.setup.php';
+		parent::setUp();
+	}
+	/**
+	 * @dataProvider FormingWikiTopicProvider
+	 */
+	public function testFormingWikiTopic( $sitename, $expectedWikiTopic ) {
```

Comment:
For clarity, since this is testing a specific method, I'd name the test after that method (i.e. `testGetWikiTopic`).